examples/ac-dc-meshed/ac-dc-data/generators.csv

```diff
@@ -1,7 +1,7 @@
 name,bus,capital_cost,efficiency,marginal_cost,p_nom,p_nom_extendable,p_nom_min,carrier
-Manchester Wind,Manchester,2793.6516029328,1,0.11,80,True,100,wind
-Manchester Gas,Manchester,196.6151679691,0.3500264336,4.5323676307,50000,True,0,gas
-Norway Wind,Norway,2184.3747960912,1,0.09,100,True,100,wind
-Norway Gas,Norway,158.2512497168,0.3568363832,5.8928445406,20000,True,0,gas
-Frankfurt Wind,Frankfurt,2129.4561224763,1,0.1,110,True,100,wind
-Frankfurt Gas,Frankfurt,102.6769530076,0.3516658529,4.0863219899,80000,True,0,gas
+Manchester Wind,Manchester,2793.6516029328,1,0.11,0,True,100,wind
+Manchester Gas,Manchester,196.6151679691,0.3500264336,4.5323676307,0,True,0,gas
+Norway Wind,Norway,2184.3747960912,1,0.09,0,True,100,wind
+Norway Gas,Norway,158.2512497168,0.3568363832,5.8928445406,0,True,0,gas
+Frankfurt Wind,Frankfurt,2129.4561224763,1,0.1,0,True,100,wind
+Frankfurt Gas,Frankfurt,102.6769530076,0.3516658529,4.0863219899,0,True,0,gas
```

Comment:
For Fabians, as later described, the new functionality changes the behavior of what happens when p_nom_extendable is true.
Before when p_nom_extendable was true, the info on p_nom is basically ignored, however this is not the case in this PR: when that occurs, the extendability is enabled but using modularity.
To preserve the old behavior, we need to put p_nom = 0.

To preserve fully the old behavior, we would need a new argument es modularity=True/False, however that was not regarded a priority and we may desire not to add a new option, and the current proposed approach could make sense.

Reply:
As discussed with fabian, we shall preserve the previous p_nom behaviour.
As such, the test csv files shall be restored as in the original formulation